purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -47,28 +53,29 @@ data class PaywallData(
         /**
          * Whether the background image will be blurred (in templates with one).
          */
-        val blurredBackgroundImage: Boolean = false,
+        @SerialName("blurred_background_image") val blurredBackgroundImage: Boolean = false,
```

Comment:
Does the serializer decode the default value if it's not present?

Reply:
Yes